tests/test_PsfexPsf.py

```diff
@@ -286,6 +286,48 @@ def testPsfexDeterminerTooFewStars(self):
         with self.assertRaisesRegex(RuntimeError, "Failed to determine"):
             psf, cellSet = self.psfDeterminer.determinePsf(self.exposure, psfCandidateListShort, metadata)
 
+    def testPsfexDeterminerNoStars(self):
+        """Test the (Psfex) psfDeterminer with no stars at all."""
+        self.setupDeterminer(self.exposure)
+        metadata = dafBase.PropertyList()
+        psfCandidateListEmpty = []
+
+        with self.assertRaises(
+            lsst.meas.extensions.psfex.psfexPsfDeterminer.PsfexNoStarsError, msg="No psf candidates supplied."
+        ):
+            psf, cellSet = self.psfDeterminer.determinePsf(self.exposure, psfCandidateListEmpty, metadata)
```

Comment:
Because it's in an `assertRaises`, there's no return here, so the left side is unnecessary (and flake8 should have complained about unused variables).

Reply:
Correct, I wasn't sensitive to it as it occurred in multiple places on the `main`. I took the liberty of fixing all those cases where no return was needed.